src/client/runner/automation/playback/rclick.js

```diff
@@ -1,163 +1,165 @@
+
+        var x          = point ? point.x : this.eventArgs.point.x;
+        var y          = point ? point.y : this.eventArgs.point.y;
+        var topElement = getElementFromPoint(x, y, expectedElement);
+
+        return {
+            point:   point || this.eventArgs.point,
+            options: options || this.eventArgs.options,
+            element: topElement || this.element
+        };
+    }
+
+    _move ({ element, offsetX, offsetY }) {
+        var moveOptions = new MoveOptions();
+
+        moveOptions.offsetX   = offsetX;
+        moveOptions.offsetY   = offsetY;
+        moveOptions.modifiers = this.modifiers;
+
+        var moveAutomation = new MoveAutomation(element, moveOptions);
+
+        return moveAutomation
+            .run()
+            .then(() => delay(automationSettings.ACTION_STEP_DELAY));
+    }
+
+    _mousedown () {
+        return cursor
+            .rightButtonDown()
+            .then(() => {
+                this.eventArgs = this._calculateEventArguments();
+
+                var isBodyElement         = this.eventArgs.element.tagName.toLowerCase() === 'body';
+                var isContentEditable     = domUtils.isContentEditableElement(this.eventArgs.element);
+                var isContentEditableBody = isBodyElement && isContentEditable;
+                var activeElement         = domUtils.getActiveElement();
+
+                // NOTE: in IE focus is not raised if element was focused
+                // before click, even if focus is lost during mousedown
+                this.eventState.simulateFocus = !browserUtils.isIE || activeElement !== this.eventArgs.element ||
+                                                isContentEditableBody;
+
+                this.eventState.simulateDefaultBehavior = eventSimulator.mousedown(this.eventArgs.element,
+                    this.eventArgs.options);
+            })
+            .then(() => this._focus());
+    }
+
+    _focus () {
+        if (this.simulateDefaultBehavior === false)
+            return nextTick();
+
+        // NOTE: For contentEditable elements we should call focus directly for action's element because
+        // option 'caretPos' is indicated for this element and elementForEvent may be a child of this element
+        var elementForFocus = domUtils.isContentEditableElement(this.element) ?
+                              this.element : this.eventArgs.element;
+
+        return automationUtil
+            .focusAndSetSelection(elementForFocus, this.eventState.simulateFocus, this.caretPos)
+            .then(() => nextTick());
+    }
+
+    _mouseup () {
+        return cursor
+            .rightButtonDown()
+            .then(() => {
+                this.eventArgs = this._calculateEventArguments();
+
+                eventSimulator.mouseup(this.eventArgs.element, this.eventArgs.options);
+
+                return delay(automationSettings.ACTION_STEP_DELAY);
+            });
+    }
+
+    _contextmenu () {
```

Comment:
I was asking what `_contextmenu` method do. It doesn't contain verb in it's name and doesn't even imply any action to me.